content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -25,9 +25,11 @@ The developer preview version provides a preview of the upcoming changes to the
   - [**threads_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#threads_deleted)
 - The **List Archives** method has a new filter, `event_types`. It replaces the `events` filter.
 - Multiple methods no longer require the `chats--my:ro`, `chats--my:rw`, `chats.conversation--my:rw`, and `chats.conversation--my:rw` scopes.
+- The **Start Chat** ([Web](/messaging/agent-chat-api/v3.4/#start-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#start-chat)) and **Resume Chat** ([Web](/messaging/agent-chat-api/v3.4/#resume-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#resume-chat)) methods now require that authors (except the requester) of all initial events are listed in the `users` field. The `visibility` of both events and users is assumed to be `all`.
```

Comment:
> The `visibility` of both events and users is assumed to be `all`.

Is it assumed to be `all` unless specified otherwise? That's kind of a mysterious sentence construction :D

Reply:
> I also noticed that we still have `recipients` in the **sample file in response** snippets in Events: https://developers.labs.livechat.com/docs/messaging/agent-chat-api/v3.4/#events
> 
> will we be updating those as well?

yes

Reply:
>Is it assumed to be all unless specified otherwise?
yes
>That's kind of a mysterious sentence construction :D
just like my English, mysterious